content/altinn-studio/guides/development/signing/runtime-delegated-signing/backend-manual/signee-provider.en.md

```diff
@@ -9,33 +9,28 @@ The implementation must return a set of individuals and/or organizations that sh
 The `Id` property in this implementation must match the ID specified in <altinn:signeeProviderId>.
 
 ```csharp
-using System;
+#nullable enable
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
-using Altinn.App.Core.Features.Signing.Interfaces;
-using Altinn.App.Core.Features.Signing.Models;
-using Altinn.App.Core.Internal.Data;
-using Altinn.App.Core.Models;
+using Altinn.App.Core.Features;
+using Altinn.App.Core.Features.Signing;
 using Altinn.App.Models.Skjemadata;
 using Altinn.Platform.Storage.Interface.Models;
 
 namespace Altinn.App.logic;
 
 public class FounderSigneesProvider : ISigneeProvider
 {
-    private readonly IDataClient _dataClient;
-
-    public FounderSigneesProvider(IDataClient dataClient)
-    {
-        _dataClient = dataClient;
-    }
-
     public string Id { get; init; } = "founders";
 
-    public async Task<SigneeProviderResult> GetSigneesAsync(Instance instance)
+    public async Task<SigneeProviderResult> GetSigneesAsync(GetSigneesParameters parameters)
```

Comment:
Kind of the wrong PR to comment but did we decide to ditch `Async` suffix?

Reply:
If we agreed on that I can remove it.

Reply:
https://github.com/Altinn/app-lib-dotnet/pull/1302